### PR TITLE
Use new data manager capabilities

### PR DESCRIPTION
**Pull request by smoufa**
See readme

### CODE REVIEW

**Comment by smoufa** [comment]
- Removed addresses and other objects from data manager groups
- Added new roles and permissions
- Cleanup of code

File: interceptors/CrmDataInterceptor.cfc [hunk 1]
```diff
@@ -42,18 +42,6 @@ component {
         }
     }
 
-    private void function _maybeEnsureUniquePrimaryAddress( interceptData ) {
```
Comment (jjannek) [inline interceptors/CrmDataInterceptor.cfc:45]: It's not clear why this function has been removed - it is still in use. Looks not correct to me.

File: views/admin/layout/sidebar/crmDataManager.cfm [hunk 2]
```diff
@@ -1,17 +1,17 @@
 <cfscript>
     subMenuItems = [];
-    if ( hasCmsPermission( "organisationmanager.read" ) ) {
+    if ( hasCmsPermission( permissionKey="crm.organisation.navigate" ) ) {
         subMenuItems.append( {
-              link  = event.buildAdminLink( linkTo="organisationManager" )
+              link  = event.buildAdminLink( objectName="organisation" )
             , title = translateResource( "cms:crmDataManager.organisations" )
-            , active = ReFindNoCase( "\.?organisationManager$", event.getCurrentHandler() )
+            , active = ReFindNoCase( "^admin\.datamanager", event.getCurrentEvent() ) && ( prc.objectName ?: "" ) == "organisation"
```
Comment (jjannek) [inline views/admin/layout/sidebar/crmDataManager.cfm]: Does this work correctly when viewing/adding/editing an organisation address / link / note?

File: views/admin/layout/sidebar/crmDataManager.cfm [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
         } );
     }
-    if ( hasCmsPermission( "personmanager.read" ) ) {
+    if ( hasCmsPermission( permissionKey="crm.person.navigate" ) ) {
         subMenuItems.append( {
-              link  = event.buildAdminLink( linkTo="personManager" )
+              link  = event.buildAdminLink( objectName="person" )
             , title = translateResource( "cms:crmDataManager.persons" )
-            , active = ReFindNoCase( "\.?personManager$", event.getCurrentHandler() )
+            , active = ReFindNoCase( "^admin\.datamanager", event.getCurrentEvent() ) && ( prc.objectName ?: "" ) == "person"
```
Comment (jjannek) [inline views/admin/layout/sidebar/crmDataManager.cfm]: see above - does this work when viewing/adding/editing a person address / link / note?

**Review by jjannek — CHANGES_REQUESTED** [review]
Great work. Only 2 minor things you might need to look at - see comments.

File: interceptors/CrmDataInterceptor.cfc (hunk 1 above)
Comment (smoufa) [reply interceptors/CrmDataInterceptor.cfc:45]: The function existed twice. I only removed the duplicate.

File: views/admin/layout/sidebar/crmDataManager.cfm (hunk 2 above)
Comment (smoufa) [reply views/admin/layout/sidebar/crmDataManager.cfm]: Fixed

File: views/admin/layout/sidebar/crmDataManager.cfm (hunk 3 above)
Comment (smoufa) [reply views/admin/layout/sidebar/crmDataManager.cfm]: Fixed

**Review by jjannek — APPROVED** [review]
Great work, thanks!

**Pull request merged**